yarn.lock

```diff
@@ -27,7 +27,7 @@
 
 "@analytics/google-analytics@^0.5.3":
   version "0.5.3"
-  resolved "https://registry.npmjs.org/@analytics/google-analytics/-/google-analytics-0.5.3.tgz"
+  resolved "https://registry.yarnpkg.com/@analytics/google-analytics/-/google-analytics-0.5.3.tgz#08642f6e9ab9a76aceb53356ea66c53552eec9b7"
```

Comment:
Any idea why the URL are changed?

Reply:
Not sure, I had done a force update of dependencies. yarnpkg is a valid domain.